### PR TITLE
Sample record inserted into db. Need to review schema

### LINKED ISSUE #1: Initial FDA Scraping Script
- Pull FDA drug label (script)
- First pass ETL
- Insert record into DB

### PR DESCRIPTION
**Pull request by kenbr7613**
Sample from https://www.accessdata.fda.gov/spl/data/bb8ef313-35e5-db13-e053-2a95a90a4306/bb8ef313-35e5-db13-e053-2a95a90a4306.xml

### CODE REVIEW

**Comment by kenbr7613** [comment]
Can mimic functionality by running
```
bash build_and_connect_docker.sh
bash setup.sh
```

File: data/scripts/dockerfiles/database_setup.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE IF NOT EXISTS drug_label_explorer;
+
+USE drug_label_explorer;
+
+CREATE TABLE IF NOT EXISTS drug_labels (
```
Comment (kenbr7613) [inline data/scripts/dockerfiles/database_setup.sql:5]: @llandau I wonder how similar of a schema you came to with EU data

**Comment by llandau** [comment]
A few notes:

1. I'm confused about the plan for the db in the docker container
2. I imagine there is a python script that downloads the xml file and inserts it into the db. I don't see that.
3. I think in Django, we create the schema by declaring one or more Model Classes (https://docs.djangoproject.com/en/4.0/intro/tutorial02/#creating-models). I don't see that.
4. It might be nice to have at least one test in Django to prove that Django can access the data (https://docs.djangoproject.com/en/4.0/intro/tutorial05/)
5. I don't like the idea of "side loading" the db from csv files. What's the process that creates the csv's? Is there a reason we want to have `sample_record.csv` checked into git? Where are the other files expected to live? How are they structured (histories, regions)? How much preprocessing is needed to convert the xml files to csv? We might want to do "other things" at that step too -- like join to the MedDRA data. Or is that going to be a separate process? Maybe it makes sense to have  both steps in the same process?
6. It looks like there is going to be a lot in the `label` field. We might need/want to break that up.
7. I haven't actually "come up with" the schema for the EU data yet. For Sprint 1, I worked on #2. I'll plan to start looking at the EU data in Sprint 2.
8. I suggest we discuss, perhaps Saturday afternoon will work, after our regular weekly meeting

File: data/scripts/dockerfiles/database_setup.sql (hunk 1 above)
Comment (llandau) [reply data/scripts/dockerfiles/database_setup.sql:5]: see comment below

Comment (kenbr7613) [reply data/scripts/dockerfiles/database_setup.sql:5]: Thanks for the quick feedback! Some more context
1. Docker + DB is to test locally. Whether or not the script is used for production deployment we can decide, but this allows the building of Maria DB to confirm that the Schema defined in create table is bug free.
2. Yes, I did not push that up as it is still WIP until the schema is finalized.
3. No I created the schema in the `database_setup.sql`file. We can add a Django Model once the server is up and running.
4. I agree. That is why I wrote all the scripts needed to build a database and insert a record. Are you doing this work?
5. The CSV is just for testing.
6. How? The point of this is to define the schema, and I do not think there is any clear indication for how we should break up the label field, so this is what I am suggesting.
7. Ok. I will move forward assuming this is the schema we will use and adjust later if needed.
8. Saturday midday I am free, Saturday evening I am not. I am also free Sunday morning/midday. We can discuss tomorrow.

**Pull request closed without merging**